packages/feature-flags/src/prompts/selectExistingEnvironment.js

```diff
@@ -0,0 +1,29 @@
+const inquirer = require('inquirer');
+
+module.exports = async function selectExistingEnvironment(environments) {
+    const { proceed } = await inquirer.prompt({
+        type: 'confirm',
+        name: 'proceed',
+        message: `
+💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥💥
+Removing or modifying existing environments WILL break any production environments using this configuration.
```

Comment:
:nit: Given we only use this prompt when deleting the environment, would it make more sense for this to be `selectEnvironmentForDeletion` and have the wording here changed to remove the reference to 'modifying existing environments' in this description?